vcf/model.py

```diff
@@ -117,6 +117,22 @@ def is_het(self):
             return None
         return self.gt_type == 1
 
+    @property
+    def is_filt(self):
```

Comment:
I don't think we have to abbreviate this and feel that `is_filtered` might be a better name, what do you think?